### PR TITLE
Add 'greater than/less than or equal to' recognizer operators

### PR DESCRIPTION
**Pull request by timgerstel**
This pull request implements the following features:
- Add 'greater than/less than or equal to' recognizer operators
- Update unit test

Signed-off-by: Timothy Gerstel <tgerstel@rocketsoftware.com>

### CODE REVIEW

**Review by 1000TurquoisePogs — APPROVED** [review]

File: base/src/dispatcher/recognizer.spec.ts [hunk 1]
```diff
@@ -125,6 +125,29 @@ describe('Recognizer', () => {
...
+      const context = { a: 456 };
+      expect(property.match(context)).to.true;
+    });
+
+    it(`should return true for GTE (type=string)`, () => {
+      const property = new RecognizerProperty('GTE', 'a', "abb");
+      const context = { a: "abc" };
+      expect(property.match(context)).to.true;
+    });
+    it(`should return true for less than (type=string)`, () => {
```
Comment (DivergentEuropeans) [inline base/src/dispatcher/recognizer.spec.ts:145]: test says it's checking for less than but we check LTE here

**Comment by lchudinov** [comment]
Usually, programming languages use 2 letter shorthands for relational operators:
- EQ Equal
- NE Not equal
- LE Less than or equal
- LT Less than
- GE Greater than or equal
- GT Greater than

I think we should follow the same convention.

File: base/src/dispatcher/dispatcher.ts [hunk 2]
```diff
@@ -1122,6 +1126,12 @@ export class RecognizerProperty extends RecognitionClause {
           case 'GT':
             super(RecognitionOp.PROPERTY_GT);
             break;
+          case 'LTE':
```
Comment (lchudinov) [inline base/src/dispatcher/dispatcher.ts:1129]: `LE`

File: base/src/dispatcher/dispatcher.ts [hunk 3]
```diff
@@ -1122,6 +1126,12 @@ export class RecognizerProperty extends RecognitionClause {
...
+            super(RecognitionOp.PROPERTY_LE);
+            break;
+          case 'GTE':
```
Comment (lchudinov) [inline base/src/dispatcher/dispatcher.ts:1132]: `GE`

**Review by lchudinov — CHANGES_REQUESTED** [review]
Please finish with renaming to make our tests pass.

**Review by lchudinov — APPROVED** [review]
Looks good to me now.

**Pull request merged**